php/libraries/OutputWrapper.class.inc

```diff
@@ -39,16 +38,26 @@ class OutputWrapper
     /**
      * Create a new command line helper.
      *
-     * @param ?SPLFileInfo $logDirectory The directory that will contain log
-     *                                   files written by this script.
+     * @param ?SPLFileInfo $logDirectory The full path to the directory that
+     *                                   will contain log files written by this
+     *                                   script.
      *
      * @return void
      */
     public function __construct(?SPLFileInfo $logDirectory = null)
     {
-        $logDir = new SPLFileInfo(
-            $logDirectory ?? \NDB_Factory::singleton()->config()->getSetting('log')
-        );
+        $config = \NDB_Factory::singleton()->config();
+        $logDir = $logDirectory ?? $config->getSetting('log');
+
+        // If the log directory is a partial path, try appending it to the
+        // LORIS base directory.
+        if (!strpos($logDir, '/') === 0) {
```

Comment:
strpos($logDir, '/') !== 0
example: tools/logs

Reply:
Yes but the code within this if statement creates `base` + `log`, so in your exampe `/var/www/loris/tools/logs` which should work.

Reply:
ex, !strpos($logDir, '/') will return !5, then !5 === 0 , so it will never go into the if condition.
 strpos($logDir, '/') !== 0 it will work.